example-notebooks/sparklines-svg-computing.ipynb

```diff
@@ -18,7 +18,9 @@
       "source": [
         "from vdom import *\n",
         "from vdom.svg import *\n",
-        "from functools import reduce"
+        "from functools import reduce\n",
+        "import random\n",
+        "import time",
```

Comment:
I made a follow on commit to delete this extraneous comma. JupyterLab fails to load at this line.